laundry-management-service/app/src/main/resources/application-docker.properties

```diff
@@ -0,0 +1,2 @@
+spring.datasource.url=jdbc:postgresql://postgres:5432/laundry_db
```

Comment:
please add comprehensive comments to all of properties. You can either take it from original file or write by yourself

Reply:
added